documentation/src/main/asciidoc/user_guide/functional_api.adoc

```diff
@@ -681,6 +681,18 @@ Consumer<WriteEntryView<String>> function =
 CompletableFuture<Void> writeFuture = writeOnlyMap.eval("key1", function);
 ----
 
+Since version 9.1 Infinispan provides overloads for all functional methods that make lambdas
+passed directly to the API serializable by default; the compiler automatically selects
+this overload if that's possible. Therefore you can call
+
+[source,java]
+----
+WriteOnlyMap<String, String> writeOnlyMap = ...
+CompletableFuture<Void> writeFuture = writeOnlyMap.eval("key1", wv -> wv.set("one"));
+----
+
+without doing the cast described above.
```

Comment:
I'd put this example first, and change the other example to use `SerializableConsumer` so that the explicit cast isn't needed.